### PR TITLE
tweaks and removes some polaris clothing, replaces with our own

### PR DESCRIPTION
**Pull request by alsoandanswer**
title. i'll get the rest sorted out soon enough

also gutted some superfluous/non-regulation clothing.

  - imageadd: "Fixes shading and missing inhands for high skirt. Adds skater skirt."
  - rscdel: "Removes some clothing from Polaris."
  - imageadd: "Beanies and some acessories have been resprited."
  - rscadd: "Adds cropped shirts and dress shirts, as well as a blouse."

Anyway. Stuff removed can be re-created with the new stuff I've added. For example, recolor the hairband to get the red one back.  Recolor the default hairpin for the pink hairpin. Want the crop dress back? Wear the skater skirt and cropped t-shirt. Hell, mix and match. You'll have more choice than before.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 1]
```diff
@@ -64,19 +64,10 @@
 	gear_tweaks += new/datum/gear_tweak/path(hazard)
 
 /datum/gear/suit/hoodie
-	display_name = "hoodie selection"
+	display_name = "hoodie"
 	path = /obj/item/clothing/suit/storage/hooded/wintercoat/hoodie
 	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
-/datum/gear/suit/hoodie/New()
```
Comment (Snakebittenn) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:71]: These were added less than a month ago (the standard period where you can revert stuff), I do not see the reasoning for taking them out.

File: html/changelogs/wezzy_polariskill.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - imageadd: "Fixes shading and missing inhands for high skirt. Adds skater skirt."
+  - rscdel: "Removes some clothing from Polaris."
+  - imageadd: "Beanies and some acessories have been resprited."
+  - rscadd: "Adds cropped shirts and dress shirts, as well as a blouse."
```
Comment (Snakebittenn) [inline html/changelogs/wezzy_polariskill.yml:44]: Where's the cropped shirt?

File: code/modules/clothing/head/misc.dm [hunk 3]
```diff
@@ -443,13 +434,6 @@
 	desc = "A head-hugging brimless winter cap. This one is tight."
 	icon_state = "beanie"
 	item_state = "beanie"
-
```
Comment (Snakebittenn) [inline code/modules/clothing/head/misc.dm:446]: This was kind of good imo, slouchy beanies do look distinct from normal ones.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 1 above)
Comment (Snakebittenn) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm:71]: This is still not past the revert period, and still lacks a reasoning.

**Comment by Snakebittenn** [comment]
The hoodies are still not past the revert period, and still lack a reasoning for removal.

**Comment by TheStryker** [comment]
I can understand cleaning things up, but the hoodies are not Polaris-ported. I've seen the crop dress be used frequently. If we remove these sprites, can you offer any replacements?

On that matter I'd also be willing to work with you on any replacements for the removed sprites since we are losing things.

**Comment by alsoandanswer** [comment]
> I can understand cleaning things up, but the hoodies are not Polaris-ported. I've seen the crop dress be used frequently. If we remove these sprites, can you offer any replacements?
> 
> On that matter I'd also be willing to work with you on any replacements for the removed sprites since we are losing things.

If you actually looked at my PR, you would've noticed I've literally added crop tops and more dresses to address this issue

**Comment by TheStryker** [comment]
> If you actually looked at my PR, you would've noticed I've literally added crop tops and more dresses to address this issue

Well alright then, what about the hoodies?

**Comment by niennab** [comment]
I'm not very good at reading code but is the red ribbon being turned into a hairband? The red ribbon is integral to one of my characters and I would hate to lose it.

**Comment by Snakebittenn** [comment]
> 
> 
> > I can understand cleaning things up, but the hoodies are not Polaris-ported. I've seen the crop dress be used frequently. If we remove these sprites, can you offer any replacements?
> > On that matter I'd also be willing to work with you on any replacements for the removed sprites since we are losing things.
> 
> If you actually looked at my PR, you would've noticed I've literally added crop tops and more dresses to address this issue

What about the hoodies?

**Comment by TheDocOct** [comment]
Feedback thread, please.

**Comment by alsoandanswer** [comment]
> I'm not very good at reading code but is the red ribbon being turned into a hairband? The red ribbon is integral to one of my characters and I would hate to lose it.
> Also is the change colourable?

Yes. 

Hairband is literally same as red ribbon. 

Just recolor it red and you can get it back.

**Comment by alsoandanswer** [comment]
> > > I can understand cleaning things up, but the hoodies are not Polaris-ported. I've seen the crop dress be used frequently. If we remove these sprites, can you offer any replacements?
> > > On that matter I'd also be willing to work with you on any replacements for the removed sprites since we are losing things.
> > 
> > 
> > If you actually looked at my PR, you would've noticed I've literally added crop tops and more dresses to address this issue
> 
> What about the hoodies?

Removed in the same vein as the keyhole sweater.

**Comment by alsoandanswer** [comment]
> > If you actually looked at my PR, you would've noticed I've literally added crop tops and more dresses to address this issue
> 
> My bad, what about the hoodies which aren't ports?

Removed in the same vein as keyhole sweater.

**Comment by Snakebittenn** [comment]
How are they similar, though?

**Comment by fernerr** [comment]
Further feedback should go in a forum thread. DNM until a month has passed since some of these items were originally implemented.

**Comment by alsoandanswer** [comment]
https://forums.aurorastation.org/topic/14572-policy-update-on-loadout-items/?tab=comments#comment-137818

Just trying to follow this thread here

**Comment by Snakebittenn** [comment]
I don't mean to dogpile or anything but it's really not cool how vague you're being, and how much you're not answering questions.

**Pull request closed without merging**